app/javascript/components/patient/Patient.js

```diff
@@ -68,665 +111,617 @@ class Patient extends React.Component {
+                      </Col>
                     )}
+                    {showMonitoredAddress && (
+                      <Col sm={showDomesticAddress && showMonitoredAddress ? 12 : 24} className="item-group">
+                        <p className="subsection-title">Monitoring Address</p>
+                        <div>
+                          <b>Address 1:</b> <span>{this.props.details.monitored_address_line_1 || '--'}</span>
+                        </div>
+                        <div>
+                          <b>Address 2:</b> <span>{this.props.details.monitored_address_line_2 || '--'}</span>
+                        </div>
+                        <div>
+                          <b>Town/City:</b> <span>{this.props.details.monitored_address_city || '--'}</span>
+                        </div>
+                        <div>
+                          <b>State:</b> <span>{this.props.details.monitored_address_state || '--'}</span>
+                        </div>
+                        <div>
+                          <b>Zip:</b> <span>{this.props.details.monitored_address_zip || '--'}</span>
+                        </div>
+                        <div>
+                          <b>County:</b> <span>{this.props.details.monitored_address_county || '--'}</span>
+                        </div>
+                      </Col>
+                    )}
+                  </Row>
+                )}
+                {(showForeignAddress || showForeignMonitoringAddress) && (
+                  <Row>
                     {showForeignAddress && (
-                      <React.Fragment>
-                        <Row>
-                          <Col>
-                            <b>Address 1:</b> <span>{this.props.details.foreign_address_line_1 || '--'}</span>
-                          </Col>
-                        </Row>
-                        <Row>
-                          <Col>
-                            <b>Address 2:</b> <span>{this.props.details.foreign_address_line_2 || '--'}</span>
-                          </Col>
-                        </Row>
-                        <Row>
-                          <Col>
-                            <b>Town/City:</b> <span>{this.props.details.foreign_address_city || '--'}</span>
-                          </Col>
-                          <Col>
-                            <b>Zip:</b> <span>{this.props.details.foreign_address_zip || '--'}</span>
-                          </Col>
-                        </Row>
-                        <Row>
-                          <Col>
-                            <b>Country:</b> <span>{this.props.details.foreign_address_country || '--'}</span>
-                          </Col>
-                        </Row>
-                      </React.Fragment>
+                      <Col sm={showForeignAddress && showForeignMonitoringAddress ? 12 : 24} className="item-group">
+                        <p className="subsection-title">Home Address</p>
```

Comment:
This should say "Foreign Address"

Reply:
Previously we had the "Home Address" display the domestic home or foreign home (whichever was there) so I'm not sure we want to change this

Reply:
might be a question for the PO team

Reply:
Decided over slack to label Home Address with (USA) or (Foreign) but leave Monitoring Address as is.  Ultimately a bigger problem with two monitoring address fields that we will need to resolve in the future